test/tpredictMaskedToken.m

```diff
@@ -0,0 +1,49 @@
+classdef(SharedTestFixtures={
+        DownloadBERTFixture, DownloadJPBERTFixture}) tpredictMaskedToken < matlab.unittest.TestCase
+    % tpredictMaskedToken   Unit test for predictMaskedToken
+    
+    % Copyright 2023 The MathWorks, Inc.
+
+    properties(TestParameter)
+        AllModels = {"base","multilingual-cased","medium",...
+            "small","mini","tiny","japanese-base",...
+            "japanese-base-wwm"}
```

Comment:
Given the tokenizers are the same for most the BERTs I think it's fine to just test "tiny" and one of the Japanese BERTs.